technic_worldgen/rubber.lua

```diff
@@ -92,7 +128,7 @@ if technic.config:get_bool("enable_rubber_tree_generation") then
 				y = (maxp.y - minp.y) / 2 + minp.y,
 				z = (maxp.z - minp.z) / 2 + minp.z}
 		local pos = minetest.find_node_near(tmp, maxp.x - minp.x,
-				{"default:dirt_with_grass"})
+				{"default:dirt_with_grass", "mcl_core:dirt_with_grass"})
```

Comment:
This would probably be better with load time selection instead of runtime, not sure if this has any measurable effect but still mcl stuff shouldn't be in world if mcl isn't going to be loaded:
```suggestion
				{has_mcl and "mcl_core:dirt_with_grass" or "default:dirt_with_grass"})
```
(could also add variable for `dirt`, similar to `lava` in oregen.lua

Reply:
done